internal/commands/run.go

```diff
@@ -158,8 +158,8 @@ func newRunCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&containerBackend, "container-backend", containerbackend.BackendAuto,
 		fmt.Sprintf("container engine backend to use, either \"%s\", \"%s\" or \"%s\"",
 			containerbackend.BackendDocker, containerbackend.BackendPodman, containerbackend.BackendAuto))
-	cmd.PersistentFlags().BoolVar(&containerPull, "container-pull", false,
-		"force pull the images before starting containers")
+	cmd.PersistentFlags().BoolVar(&containerLazyPull, "container-lazy-pull", false,
+		"when starting containers only pull images that are not available locally")
```

Comment:
Maybe

```suggestion
		"attempt to pull images only if they are missing locally (helpful in case of registry rate limits)")
```

To highlight there otherwise there will be not even an attempt to pull?